app/forms/steps/claim_reason_form.rb

```diff
@@ -0,0 +1,48 @@
+require 'steps/base_form_object.rb'
+
+module Steps
+  class ClaimReasonForm < Steps::BaseFormObject
+    attribute :claim_reason, :value_object, source: ClaimReason
```

Comment:
so the claim reasons are a series of checkbox with no limit on how many can be selected at once. As such I think we will need/want a field for each checkbox (note some checkboxes then have an additional data requirement)

Reply:
will this be in the Claim table or a seperate table ? will they be of type bool